bundles/org.openhab.binding.pioneeravr/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -198,6 +198,8 @@
 			<channel id="volumeDb" typeId="volumeChannelDb" />
 			<channel id="mute" typeId="muteChannel" />
 			<channel id="setInputSource" typeId="setInputSourceChannel" />
+			<channel id="ListeningMode" typeId="ListeningModeChannel" />
```

Comment:
All other channels start with a lower case character:
```suggestion
			<channel id="listeningMode" typeId="ListeningModeChannel" />
```

Reply:
You are totally right. Nice eyes, indeed!

Reply:
You need to do it at all places where this channel and typeid is used. Not only here. I just put it here as reference :smile:

Reply:
Done in https://github.com/openhab/openhab2-addons/pull/5515/commits/c3207e4c94bca224ae4759810280959207285614